src/management/ActionsManager.js

```diff
@@ -53,21 +53,21 @@ var ActionsManager = function(options) {
    * @type {external:RestClient}
    */
   var auth0RestClient = new Auth0RestClient(
-    options.baseUrl + '/actions/actions/:action_id',
+    options.baseUrl + '/actions/actions/:id',
```

Comment:
Hey @stevezau! This is a great catch, but since this was released in the latest stable version of node-auth0 I think we'll need to make this change in a backwards compatible way (in other words, making sure all of these functions accept either `action_id` or `id`.